packages/framework/src/Controller/Admin/SeoController.php

```diff
@@ -3,40 +3,31 @@
 namespace Shopsys\FrameworkBundle\Controller\Admin;
 
 use Shopsys\FrameworkBundle\Component\Domain\AdminDomainTabsFacade;
+use Shopsys\FrameworkBundle\Form\Admin\Seo\SeoRobotsSettingFormType;
 use Shopsys\FrameworkBundle\Form\Admin\Seo\SeoSettingFormType;
 use Shopsys\FrameworkBundle\Model\Seo\SeoSettingFacade;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
 class SeoController extends AdminBaseController
 {
-    /**
-     * @var \Shopsys\FrameworkBundle\Component\Domain\AdminDomainTabsFacade
-     */
-    protected $adminDomainTabsFacade;
-
-    /**
-     * @var \Shopsys\FrameworkBundle\Model\Seo\SeoSettingFacade
-     */
-    protected $seoSettingFacade;
-
     /**
      * @param \Shopsys\FrameworkBundle\Model\Seo\SeoSettingFacade $seoSettingFacade
      * @param \Shopsys\FrameworkBundle\Component\Domain\AdminDomainTabsFacade $adminDomainTabsFacade
      */
     public function __construct(
-        SeoSettingFacade $seoSettingFacade,
-        AdminDomainTabsFacade $adminDomainTabsFacade
+        protected readonly SeoSettingFacade $seoSettingFacade,
+        protected readonly AdminDomainTabsFacade $adminDomainTabsFacade,
     ) {
-        $this->seoSettingFacade = $seoSettingFacade;
-        $this->adminDomainTabsFacade = $adminDomainTabsFacade;
     }
 
     /**
      * @Route("/seo/")
      * @param \Symfony\Component\HttpFoundation\Request $request
+     * @return \Symfony\Component\HttpFoundation\Response
      */
-    public function indexAction(Request $request)
+    public function indexAction(Request $request): Response
```

Comment:
adding a return type is a BC-break. please remove it.

see https://symfony.com/doc/current/contributing/code/bc.html

Reply:
Removed.